app/components/works/dates_component.html.erb

```diff
@@ -1,176 +1,170 @@
-              class: "form-control", min: min_year, max: max_year %>
-          <div class="invalid-feedback" 
-            data-date-validation-target="error"
-            data-date-clear-target="error"></div>
-        </div>
-        <div class="month">
-          <label for="<%= prefix %>_created_month" class="visually-hidden">Created month</label>
-          <%= select_month created_month, { prefix: prefix, field_name: 'created(2i)', prompt: 'month'},
-                          data: { date_validation_target: 'month',
-                                  date_clear_target: 'month',
-                                  action: 'date-validation#change' },
-                          id: "#{prefix}_created_month", class: "form-control" %>
-        </div>
-
-        <div class="day">
-          <label for="<%= prefix %>_created_day" class="visually-hidden">Created day</label>
-          <%= select_day created_day, { prefix: prefix, field_name: 'created(3i)', prompt: 'day'},
-                        data: { date_validation_target: 'day',
-                          date_approximate_target: 'day',
-                          date_clear_target: 'day',
-                          action: 'date-validation#change date-approximate#change' },
-                        id: "#{prefix}_created_day", class: "form-control" %>
-        </div>
-        <%= button_tag type: 'button', class: 'btn btn-sm ms-3', aria: { label: "Clear created date" },
-                     data: { action: "date-clear#clear" } do %>
-          <span class="fa-regular fa-trash-alt"></span>
-        <% end %>
-        <div>
-        <%= check_box_tag "#{prefix}[created(approx0)]", true, created_approximate?,
-          data: { date_approximate_target: 'approximate',
-            date_clear_target: 'approximate',
-            action: 'date-approximate#change' },
-          class: 'form-check-input' %>
-        <%= form.label 'created(approx0)', 'Approximate (can\'t be selected if \'day\' is entered)' %>
+      <div class="col-sm-10">
+        <div class="row ms-1">
+          <div class="col form-check">
+            <%= form.check_box :created_type, { class: "form-check-input", data: {"complex-checkbox-target": "checkbox", action: "complex-checkbox#toggleInputs"} }, 'range', 'single' %>
+            <%= form.label :created_type, 'Enter a date range', class: "form-check-label" %>
+          </div>
         </div>
-      </div>
-    </div>
-
-
-    <div class="mb-3 row" data-complex-radio-target="selection">
-      <div class="col-md-2">
-        <%= form.radio_button :created_type, 'range', class: 'form-check-input', data: { action: 'complex-radio#disableUnselectedInputs' } %>
-        <%= form.label :created_type_range, 'Date range', class: 'form-check-label fw-semibold' %>
-      </div>
+        <div class="row">
+          <div data-controller="date-validation date-approximate date-clear" 
+            data-date-approximate-enabled-value="<%= form.object.created_type == 'single' %>" 
```

Comment:
Can this be moved to `single?` in the Works::DateComponent?

Reply:
Done.